atomicapp/providers/openshift.py

```diff
@@ -179,15 +266,7 @@ def _process_template(self, template):
         """
         logger.debug("processing template: %s", template)
         url = self._get_url(self._get_namespace(template), "processedtemplates")
-        (status_code, return_data) = self._make_request("post", url, data=template)
-        if status_code == 201:
-            logger.info("template proccessed %s", template['metadata']['name'])
-            logger.debug("processed template %s", return_data)
-            return return_data['objects']
-        else:
-            msg = "%s %s" % (status_code, return_data)
-            logger.error(msg)
-            raise ProviderFailedException(msg)
+        return self.oc.process_template(url, template)
```

Comment:
The reason I did not remove ``_process_template`` is because of the dependency on ``get_url``. Also, I see no reason to move ``get_url`` to ``OpenshiftClient``.